Makefile

```diff
@@ -1,4 +1,4 @@
 std:
-	g++ test.cc -o test -DUSE_STD_MAP
+	g++ test.cc stdmap.cpp -o test -DUSE_STD_MAP
```

Comment:
Let's keep the naming convention the same, also we can add flags to make it easier to compile with new files. What you have works for now though.